### PR TITLE
Add vertical scroll on Footer menu

### PR DESCRIPTION
**Pull request by dojutsu-user**
Fixes https://github.com/rtfd/readthedocs.org/issues/4496

Result:

1. When there are too many versions and translations
ezgif-4-045218f8984e

2. When there are limited versions and translations
ezgif-4-8a20fdd2583f

### CODE REVIEW

**Comment by jessetan** [comment]
Do you have a sphinx project we can use to test this?

File: sass/_theme_badge.sass [hunk 1]
```diff
@@ -35,7 +35,8 @@
       color: $black
   &.shift-up
     height: auto
-    max-height: 100%
+    max-height: 90%
```
Comment (jessetan) [inline sass/_theme_badge.sass]: What is the reason for this line? Why not let it take 100% height?

Comment (dojutsu-user) [reply sass/_theme_badge.sass]: With `max-height: 100%`, this happens:

ezgif-3-b0f25b01dc0b

Comment (jessetan) [reply sass/_theme_badge.sass]: OK, I see. The height of 90% is fine for this example where the rtd theme is not used, but just the badge_only CSS. Here, the versions popup is not aligned to the bottom of the window.
When the rtd theme is used, the versions popup is aligned to the bottom of the window and a  height of 100% would be better.

Can you try keeping `max-height: 100%` on `&.shift-up` as it is, but setting `max-height: 90%` on `&.rst-badge` in `_theme_badge.sass:60`?

Comment (dojutsu-user) [reply sass/_theme_badge.sass]: That works.
I pushed the changes.

File: sass/_theme_badge.sass [hunk 2]
```diff
@@ -64,6 +65,7 @@
     left: auto
     border: none
     max-width: $nav-desktop-width
+    max-height: 100%
```
Comment (jessetan) [inline sass/_theme_badge.sass]: Did you mean:
```suggestion
    max-height: 90%
```
or does 100% work for you as well?

**Comment by humitos** [comment]
Just a comment.

I think there is only one difference with the current behavior (but not sure if it's browser dependent): in current versions of documentation with `alabaster` theme the vertical does not appear when scroll is not needed and in the GIF uploaded to this pull request, it does appear even when there is nothing to scroll.

captura de pantalla_2018-12-20_14-54-50

**Comment by davidfischer** [comment]
> I think there is only one difference with the current behavior (but not sure if it's browser dependent): in current versions of documentation with `alabaster` theme the vertical does not appear when scroll is not needed and in the GIF uploaded to this pull request, it does appear even when there is nothing to scroll.

If I'm understanding this correctly, you're talking about the scrollbar appearing or not. I believe this is browser dependent. Chrome allowed changing some appearance on scrollbars while Firefox did not. However, I believe that is changing.

**Pull request merged**